rfcs/2021-11-whcm-support.md

```diff
@@ -0,0 +1,108 @@
+# RFC: Windows High Contrast Mode (WHCM) Support
+
+**Date**: November 29, 2021
+**Status**: `WIP`
+
+## Background
+
+Windows High Contrast Mode (WHCM) is a unique accessibility feature originally designed for Windows users with contrast sensitivity issues. Other reasons a user might enable WHCM include:
+
+- Reducing visual noise to improve focus
+- Easing eye strain, migraines, or light sensitivity
+- Preferring a specific color scheme
+
+With WHCM, users can select a theme of colors for a scoped number of semantic elements. This theme is then applied to user interfaces and applications.
```

Comment:
I might mention how these colors are mapped to a subset of [CSS2 System Color keywords](https://www.w3.org/TR/css-color-3/#css2-system), which tie into Windows OS user interface components. This is the bit that allows web content coloration to be dynamic!